bandit/core/node_visitor.py

```diff
@@ -234,19 +238,19 @@ def generic_visit(self, node):
                 for idx, item in enumerate(value):
                     if isinstance(item, ast.AST):
                         if idx < max_idx:
-                            setattr(item, 'sibling', value[idx + 1])
+                            setattr(item, 'bandit_sibling', value[idx + 1])
```

Comment:
is there a reason these are `setattr(...)` instead of just `item.bandit_sibling = ...`

there's no safety to using `setattr` with the second argument being constant (same with 2-arg `getattr` with constant second argument)

Reply:
Not sure why they are setattr other than that is how the codebase was already setup. I would be up to make a change in another PR to update this